src/Telegram/Client.php

```diff
@@ -330,4 +331,16 @@ private function targetChatMessageOrInlineMessageId(array $opt = []): array
             'message_id' => $this->messageId(),
         ]);
     }
+
+    /**
+     * Chunk a string into an array of strings.
+     * @param  string  $text
+     * @param  int  $length
+     * @return array
+     */
+    protected function chunkText(string $text, int $length = Limits::TEXT_LENGTH): array
+    {
+        $wrapper = StringUtils::getWrapper();
+        return explode('%#TGMSG#%', $wrapper->wordWrap($text, $length, "%#TGMSG#%", true));
```

Comment:
Maybe is worth it try to do something like this: https://stackoverflow.com/questions/3825226/multi-byte-safe-wordwrap-function-for-utf-8

Reply:
Their implementation is literally the Drupal function in the stackoverflow link: https://github.com/laminas/laminas-stdlib/blob/3.11.x/src/StringWrapper/AbstractStringWrapper.php#L156

Reply:
### laminas:
![immagine](https://user-images.githubusercontent.com/4071613/179839271-d3eddd5f-3435-44e1-aabf-8c01a47b488f.png)

### stack overflow:
![immagine](https://user-images.githubusercontent.com/4071613/179839349-94bf6a17-60d3-4679-a8a2-0a20cd08dedb.png)